packages/web-components/src/components/va-search/va-search.tsx

```diff
@@ -124,37 +229,182 @@ export class VaSearch {
     this.buttonKeyDownEvent.emit(event);
   };
 
+  // Suggestions Event Handlers
+  private handleSuggestionClickEvent = (
+    event: KeyboardEvent | MouseEvent,
+    index,
+  ) => {
+    this.suggestionClickEvent.emit(event);
+    const suggestion = this.el.shadowRoot.getElementById(
+      `listbox-option-${index}`,
+    );
+    if (!suggestion) return;
+    const selectedSuggestion = this.el.shadowRoot.querySelector(
+      '[aria-selected="true"]',
+    );
+    if (selectedSuggestion) selectedSuggestion.removeAttribute('aria-selected');
+    this.inputRef.focus();
+    this.inputRef.value = suggestion.textContent;
+  };
+
+  private handleSuggestionFocusEvent = (event: FocusEvent) => {
+    this.suggestionFocusEvent.emit(event);
+  };
+
+  private handleSuggestionKeyDownEvent = (event, index) => {
+    this.suggestionKeyDownEvent.emit(event);
+
+    const options = this.el.shadowRoot.querySelectorAll('[role="option"]');
+
+    switch (event.key) {
+      case 'ArrowUp':
+        const lastOption = options[options.length - 1] as HTMLDivElement;
+        if (index === 0) {
+          if (!lastOption) return;
+          this.selectSuggestion(lastOption);
+        } else {
+          if (!options[index - 1]) return;
+          this.selectSuggestion(options[index - 1]);
+        }
+        break;
+      case 'ArrowDown':
+        const firstOption = options[0] as HTMLDivElement;
+        if (index === options.length - 1) {
+          if (!firstOption) return;
+          this.selectSuggestion(firstOption);
+        } else {
+          if (!options[index + 1]) return;
+          this.selectSuggestion(options[index + 1]);
+        }
+        break;
+      case 'Enter':
+        this.inputRef.value = options[index].textContent;
+        this.inputRef.focus();
+        break;
+      case 'Escape':
+        this.inputRef.value = '';
+        this.inputRef.focus();
+        break;
+      case 'Home':
+        this.inputRef.focus();
+        this.inputRef.setSelectionRange(0, 0);
+        break;
+      case 'ArrowRight':
+      case 'End':
+        this.inputRef.focus();
+        this.inputRef.setSelectionRange(
+          this.inputRef.value.length + 1,
+          this.inputRef.value.length + 1,
+        );
+        break;
+      case 'ArrowLeft':
+        this.inputRef.focus();
+        this.inputRef.setSelectionRange(
+          this.inputRef.value.length,
+          this.inputRef.value.length,
+        );
+        break;
+      default:
+        this.inputRef.focus();
+        this.inputRef.value = this.inputRef.value + `${event.key}`;
+        break;
+    }
+  };
+
+  private handleSuggestionMouseDownEvent = (event: MouseEvent) => {
+    this.suggestionMouseDownEvent.emit(event);
+  };
+
+  private handleSuggestionMouseOverEvent = (event: MouseEvent) => {
+    this.suggestionMouseOverEvent.emit(event);
+  };
```

Comment:
What's happening on mouseover that we need to support?

Reply:
[This](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/search/components/SearchDropdown/SearchDropdownComponent.js#L694-L696) is how it's being used in the SearchDropdownComponent in `vets-website`. I think it's safe to remove.